code/modules/goonhub/api/routes/rounds/UpdateGameRound.dm

```diff
@@ -0,0 +1,9 @@
+
+/// PUT /rounds/{gameRound}
+/// Update a game round. This should be used when game round data we care about is set after the start of the round.
+/datum/apiRoute/gameround/update
```

Comment:
this should be `rounds`..... it needs to reflect the api!

Reply:
ah I have not been doing that as a hard rule, whoops. Will update all prs to match this